.github/workflows/pr.yml

```diff
@@ -15,6 +15,11 @@ jobs:
     if: ${{ github.repository == 'ni/measurementlink-labview' || github.event.pull_request.head.repo.full_name == 'ni/measurementlink-labview' }}
     uses: ./.github/workflows/run_g_tests.yml
 
+  run_vi_analyzer:
+    name: Run VI Analyzer
+    if: ${{ github.repository == 'ni/measurementlink-labview' || github.event.pull_request.head.repo.full_name == 'ni/measurementlink-labview' }}
```

Comment:
I see this same 'if' check is done above, but do you know why we need this? Isn't the PR always run on 'this' repo, which is always ni/measurementlink-labview?